okhttp-tests/src/test/java/okhttp3/EventListenerTest.java

```diff
@@ -147,9 +203,9 @@
     } catch (IOException expected) {
     }
 
-    listener.findNextEvent(DnsStart.class);
+    assertNotNull(listener.removeUpToEvent(DnsStart.class));
```

Comment:
... at least in tests I assume we’ll always know whether an event should be present?